project_security_group/models/project.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Antiun Ingeniería S.L. - Sergio Teruel
+# (c) 2015 Antiun Ingeniería S.L. - Carlos Dauden
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import models, fields, api
+from openerp.tools.safe_eval import safe_eval
+from openerp.osv.orm import setup_modifiers
+from lxml import etree
+
+
+class Task(models.Model):
+    _inherit = "project.task"
+
+    project_members = fields.Many2many(related='project_id.members')
+
+    @api.model
+    def fields_view_get(self, view_id=None, view_type='form', toolbar=False,
+                        submenu=False):
+        res = super(Task, self).fields_view_get(
+            view_id, view_type, toolbar=toolbar, submenu=submenu)
+        if view_type == 'form':
+            res['fields'].update(
```

Comment:
@pedrobaeza @dreispt 
The intention was only show the field if only_team_user option is checked but this injection should have been inside the if statement.
This field is necessary for it use in user_id field domain.
I fix it in next commit.